client/webserver/websocket.go

```diff
@@ -82,11 +82,14 @@ func (s *WebServer) websocketHandler(conn ws.Connection, ip string) {
 	s.mtx.Unlock()
 	defer func() {
 		cl.mtx.Lock()
+		defer cl.mtx.Unlock()
 		if cl.quit != nil {
 			cl.quit()
 		}
+
```

Comment:
Might as well do `cl.mtx.Unlock()` here since it's not protecting `cl.cid`.

Reply:
`cl.mtx.Unlock()` is deferred and delete reads `ci.cid`.

Reply:
Right, but I'm saying it doesn't need to protect `ci.cid` since `ci.cid` is not written after construction of the `wsClient`.  Concurrent reads are OK.

Reply:
gotcha, rectifying.

Reply:
Whoops. Good catch.